### PR TITLE
 added minor changes

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Solving minor UI issues in the iq dashboard
- [x] FontWeight of all text to be a minimum of 500
- [x] header on stats table on both lock and staking table fixed for large screens
closes https://github.com/EveripediaNetwork/issues/issues/718

### CODE REVIEW

**Review by kesar — CHANGES_REQUESTED** [review]
https://github.com/EveripediaNetwork/iq-ui/commit/f2f4d2333ddc0b0d055102b79bc521a44283cd43#r84068604

**Comment by OleanjiKingCode** [comment]
> [f2f4d23#r84068604](https://github.com/EveripediaNetwork/iq-ui/commit/f2f4d2333ddc0b0d055102b79bc521a44283cd43#r84068604)

Feyi requested that I do some changes and that's what I was doing kesar

**Comment by kesar** [comment]
> > [f2f4d23#r84068604](https://github.com/EveripediaNetwork/iq-ui/commit/f2f4d2333ddc0b0d055102b79bc521a44283cd43#r84068604)
> 
> Feyi requested that I do some changes and that's what I was doing kesar

I see there was some missing sync, lets discuss it tomorrow and make sure we are all in the same page about the order 👍🏻

**Comment by kesar** [comment]
@Simplypheyie96 can we make sure we are all ni the same page w the IQ locking order stats? https://github.com/EveripediaNetwork/issues/issues/718#issuecomment-1248618597

let's make sure this gets solved in the PR and conflicts are solved @OleanjiKingCode

**Review by Simplypheyie96 — APPROVED** [review]

**Comment by Simplypheyie96** [comment]
@OleanjiKingCode Next thing to work on is the staking page. The data is not yet complete.
the staking table contains 2 entries for the APR data which is missing on implementation
image

Also, on larger screens, the staking table should be fully visible without the scroll bar like in the picture below.
image

**Comment by OleanjiKingCode** [comment]
@Simplypheyie96  You can check them out now

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
On smaller screens, the table will remain with a scroll, because now, the staking page no longer has the same padding on the side. The full table will only show on larger screens because there is more space.
I specified this on figma
image

**Comment by Simplypheyie96** [comment]
Also, there are still some pages with less than 500 font weight 
image

**Comment by Simplypheyie96** [comment]
Link appropriate pages for the swap page icons

**Comment by OleanjiKingCode** [comment]
> we have some meaningless and broken cards here on the voting page under old votes which I think should be removed @OleanjiKingCode image

they are coming from the backend @Simplypheyie96

**Comment by Simplypheyie96** [comment]
can we have @ashaka work on that. please help remove all the unnecessary cards on the voting page

**Comment by kesar** [comment]
thats not coming from our api but from the snapshot provider, we could manually hide them in the UI since they were low quality proposals https://snapshot.everipedia.com/

**Comment by Simplypheyie96** [comment]
Final addition (I hope) @OleanjiKingCode 
change all circled fonts to gray 600. Also align the "transferring to pIQ" part for better contrast. 
In dark mode, change circled fonts to white-alpha 900
also, all strokes for each box should be gray 300 in light mode while strokes in dark mode should be white-alpha 300 respectively
image

File: src/components/cards/PlatformCard.tsx [hunk 1]
```diff
@@ -19,9 +22,16 @@ const PlatformCard = ({
...
         borderColor="divider"
         rounded="lg"
+        _hover={{
+          bg: 'divider',
+          color: 'dimmedText',
+        }}
       >
-        <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />
+        <NextLink href={route} passHref>
+          <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />
```
Comment (anubra266) [inline src/components/cards/PlatformCard.tsx:31]: Only use `passhref` when the child is an `a` tag
```suggestion
        <NextLink href={route}>
          <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />

File: src/theme/foundations/typography.ts [hunk 2]
```diff
@@ -27,10 +27,6 @@ export const typography = {
   },
 
   fontWeights: {
-    hairline: 100,
-    thin: 200,
-    light: 300,
-    normal: 400,
```
Comment (anubra266) [inline src/theme/foundations/typography.ts:33]: Why are you getting rid of these?

**Review by anubra266 — CHANGES_REQUESTED** [review]
Left comments

File: src/theme/foundations/typography.ts (hunk 2 above)
Comment (OleanjiKingCode) [reply src/theme/foundations/typography.ts:33]: @Simplypheyie96 wanted the minimum font weight to be 500

Comment (anubra266) [reply src/theme/foundations/typography.ts:33]: Okay...but we don't need to touch these.

**Review by Simplypheyie96 — APPROVED** [review]
Awesome Job @OleanjiKingCode everything is on point now. 
No more fixes for now. 👏🏿

**Comment by kesar** [comment]
merging to show today!

**Pull request merged**